boards/nordic/nrf7120pdk/Kconfig.nrf7120pdk

```diff
@@ -6,6 +6,7 @@ config BOARD_NRF7120PDK
 						BOARD_NRF7120PDK_NRF7120_CPUAPP_EMU || \
 						BOARD_NRF7120PDK_NRF7120_CPUAPP_FPGA
 	select SOC_NRF7120_PREENG if BOARD_NRF7120PDK_NRF7120_CPUAPP_EMU || \
-						BOARD_NRF7120PDK_NRF7120_CPUAPP_FPGA
+						BOARD_NRF7120PDK_NRF7120_CPUAPP_FPGA || \
+                        BOARD_NRF7120PDK_NRF7120_CPUAPP
```

Comment:
This is wrong, `BOARD_NRF7120PDK_NRF7120_CPUAPP` is only for real silicon no PREENG